docs/05-technical-reference/ra-01-configuring-runtime.md

```diff
@@ -2,7 +2,7 @@
 title: Configuring the Runtime
 ---
 
-> **NOTE:** To represent API and Event Definitions of the Applications connected to a Runtime, we recommend that you use the Open Service Broker API.
+> **NOTE:** To represent API and Event Definitions of the Applications connected to a Runtime, we recommend that you use the OpenServiceBroker API.
```

Comment:
Are you sure OpenServiceBroker should be in camelcase?

Reply:
Nope :D But whatever we do, let's be consistent here and in https://github.com/kyma-project/kyma/pull/14991#discussion_r937691774

Reply:
This doesn't look like something that should be CamelCased. I checked in the [old Kyma documentation](https://kyma-project-old.netlify.app/docs/components/runtime-agent/#details-configuring-the-runtime) as well as their [website](https://www.openservicebrokerapi.org/) (and Google), and it's written with spaces in all of them.

Reply:
updated with https://github.com/kyma-project/kyma/pull/14991/commits/7cb968371707df499a7fdc4bb2bfb64852cc1740